src/crypto/aes.rs

```diff
@@ -302,6 +303,33 @@ mod tests {
         assert_eq!(*decrypted_plaintext, plaintext[..]);
     }
 
+    // Even very small documents of 0 and 1 bytes should roundtrip between AesEncryptedValue and bytes
+    #[test]
+    fn test_roundtrip_aesencryptedvalue_zero_one_bytes() -> Result<(), IronOxideErr> {
```

Comment:
This test name seems like it could be improved or the test changed because you're not really doing a "roundtrip" test right? I would expect a roundtrip to do an encrypt and decrypt and assert you got the same value out.

Reply:
My intention is that the `AesEncryptedValue` is roundtripping to/from a bytes representation. This is narrower than a full AES encrypt/decrypt. I only included the encrypt step to avoid referencing a the `AES_IV_LEN` and `AES_GCM_TAG_LEN` consts. That was probably a bad idea. I have reworked this test to remove encrypt so hopefully it's more clear what's being tested.